modules/standard/Random.chpl

```diff
@@ -638,6 +638,30 @@ module Random {
         const oddseed = if seed % 2 == 0 then seed + 1 else seed;
         return oddseed;
       }
+      /*
+      Generate a seed based on the entropy pool in the OS; this requires
+      the presence of /dev/urandom.  This should be standard on both macOS
+      and Linux systems.
+
+      For highly parallel applications on one locale, it may be necessary
+      to lock this object to ensure two tasks do not both pull the same
+      bit of entropy.
+      */
+      proc type devURandomSeed: int(64) {
+        use IO;
+        var seed: int(64);
+        try {
+          var entropy = open('/dev/urandom', iomode.r);
+          var entropyStream = entropy.reader();
+          entropyStream.readbits(seed, 64);
+        } catch {
+          // it would be nice to default to the timer, except
+          // we shouldn't invisibly use the timer when /dev/urandom isn't
+          // available.  So for now, this is 0.
+          // seed = oddCurrentTime;
```

Comment:
I think it would be nice to create a different one, like `devURandomOrTime`, that does invisibly switch and that could serve as the default.

Reply:
then have `devURandom` fail loudly via halt (or throw?) if `dev/urandom` is not found.

Reply:
I like this idea, too.